storage_service/locations/api/beta/remple/openapi.py

```diff
@@ -0,0 +1,1573 @@
+                    resource_name, rsrc_collection_name, pk_patt, resource_cls)
+            elif action == 'new':
+                path = get_collection_targeting_openapi_path(
+                    rsrc_collection_name, modifiers=['new'])
+            else:  # edit is default case
+                path, path_params = get_member_targeting_openapi_path(
+                    resource_name, rsrc_collection_name, pk_patt, resource_cls,
+                    modifiers=['edit'])
+            if path_params:
+                path_dict = paths.setdefault(path, {'parameters': path_params})
+            else:
+                path_dict = paths.setdefault(path, {})
+            responses_meth = '_get_{}_responses'.format(action)
+            parameters_meth = '_get_{}_parameters'.format(action)
+            request_body_meth = '_get_{}_request_body'.format(action)
+            parameters = getattr(self, parameters_meth, lambda x: None)(
+                resource_name)
+            request_body = getattr(self, request_body_meth, lambda x: None)(
+                resource_name)
+            summary, description = _summarize(
+                action, resource_name, rsrc_collection_name)
+            path_dict[http_method] = OrderedDict([
+                ('summary', summary),
+                ('description', description),
+                ('operationId', operation_id),
+            ])
+            if parameters:
+                path_dict[http_method]['parameters'] = parameters
+            if request_body:
+                path_dict[http_method]['requestBody'] = request_body
+            path_dict[http_method]['responses'] = getattr(
+                self, responses_meth)(resource_name)
+            path_dict[http_method]['tags'] = [rsrc_collection_name]
+
+    def _set_custom_paths(self, paths, resource_name, resource_cls, pk_patt,
+                          rsrc_collection_name):
+        """Set an attribute on dict ``paths`` for each custom endpoint
+        configured on ``resource_cls``.
+        """
+        for custom_endpoint in self._get_custom_endpoints(resource_cls):
+            http_method = custom_endpoint.http_method
+            operation_id = _get_operation_id(
+                custom_endpoint.action, resource_name)
+            path = '/{}/{}'.format(rsrc_collection_name, custom_endpoint.path)
+            path_dict = {
+                http_method: OrderedDict([
+                    ('operationId', operation_id),
+                    ('summary', custom_endpoint.summary),
+                    ('description', custom_endpoint.description),
+                ])
+            }
+            for attr in ('parameters', 'request_body', 'responses', 'tags'):
+                val = getattr(custom_endpoint, attr)
+                if val:
+                    path_dict[http_method][attr] = val
+            paths[path] = path_dict
+
+    def _get_search_request_body_examples(self, resource_name):
```

Comment:
This is specific to the Archivematica Storage Service so it should be removed or moved outside of Remple. Futhermore, it seems that it is not even being used.